serverless/admin_guide/install-serverless-operator.adoc

```diff
@@ -13,23 +13,6 @@ This guide walks cluster administrators through installing the {ServerlessOperat
 {ServerlessProductName} is supported for installation in a restricted network environment. For more information, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
 ====
 
-include::modules/serverless-cluster-sizing-req.adoc[leveloffset=+1]
-
-[id="installing-openshift-serverless-advanced-reqs"]
-== Additional requirements for advanced use cases
-
-For more advanced use cases such as logging or metering on {product-title}, you must deploy more resources. Recommended requirements for such use cases are 24 CPUs and 96GB of memory.
-
-If you have high availability (HA) enabled on your cluster, this requires between 0.5 - 1.5 cores and between 200MB - 2GB of memory for each replica of the Knative Serving control plane.
-HA is enabled for some Knative Serving components by default. You can disable HA by following the documentation on xref:../../serverless/admin_guide/serverless-ha.adoc#serverless-ha[Configuring high availability replicas on {ServerlessProductName}].
```

Comment:
This is valid information for sure relates to current state. I think it should be left.